datashader/glyphs.py

```diff
@@ -29,10 +29,12 @@ def validate(self, in_dshape):
             raise ValueError('y must be real')
 
     def _compute_x_bounds(self, df):
-        return df[self.x].min(), df[self.x].max()
+        xs = df[self.x].values
+        return xs.min(), xs.max()
```

Comment:
Should this and `_compute_y_bounds` be @jit so that Numba can combine the max and min into a single pass?